tests/client/components/AddFood.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { mount } from 'enzyme'
+
```

Comment:
Please remove this new line also. New line only needed between external imports and local imports. Thank you! :) 
Tests pass ✅

Reply:
done!